src/main/jbake/content/blog/2017/02/08/hawkular-metrics-0.24.0.Final-released.md

```diff
@@ -0,0 +1,78 @@
+title=Hawkular Metrics 0.24.0 - Release
+author=Stefan Negrea
+date=2017-02-08
+type=post
+tags=blog, metrics, release
+status=published
+~~~~~~
+
+I am happy to announce release 0.24.0 of Hawkular Metrics. This release is anchored by a new tag query language and general stability improvements.
+
+Here is a list of major changes:
+
+* **Tag Query Language**
+    * A query language was added to support complex constructs for tag based queries for metrics
+    * The new syntax supports:
+        * logical operators: `AND`,`OR`
+        * equality operators: `=`, `!=`
+        * value in array operators: `IN`, `NOT IN`
+        * existential conditions:
+            * tag without any operator is equivalent to `= '*'`
+            * tag preceded by the NOT operator matches only instances without the tag defined
+        * all the values in between single quotes are treated as regex expressions
+        * simple text values do not need single quotes
+        * spaces before and after equality operators are not necessary
+    * For more details please see: [Pull Request 725](https://github.com/hawkular/hawkular-metrics/pull/725), [HWKMETRICS-523](https://issues.jboss.org/browse/HWKMETRICS-523)
+    * Sample queries:
+
+```
+a1 = 'bcd' OR a2 != 'efg'
+a1='bcd' OR a2!='efg'
+a1 = efg AND ( a2 = 'hijk' OR a2 = 'xyz' )
+a1 = 'efg' AND ( a2 IN ['hijk', 'xyz'] )
+a1 = 'efg' AND a2 NOT IN ['hijk']
+a1 = 'd' OR ( a1 != 'ab' AND ( c1 = '*' ) )
+a1 OR a2
+NOT a1 AND a2
+a1 = 'a' AND NOT b2
+a1 = a AND NOT b2
+```
```

Comment:
So even *you* don't know how to do it, which proves that an example would be good.